zone/attack.cpp

```diff
@@ -3957,10 +3958,11 @@ void Mob::CommonDamage(Mob* attacker, int64 &damage, const uint16 spell_id, cons
 		if (attacker) {
 			if (skill_used == EQ::skills::SkillBash) {
 				can_stun = true;
-				if (attacker->IsClient())
+				if (attacker->IsClient() || attacker->IsBot() || attacker->IsMerc()) {
```

Comment:
This is starting to be verbose and common usage pattern

Let's create a shorthand to consolidate

```
attacker->IsOfClientBotMerc()
```

Reply:
excellent, I was going to ask you or KK about this. I'll add it.

Reply:
Can fix this, too.